src/pretix/base/plugins.py

```diff
@@ -6,6 +6,7 @@
 from django.apps import AppConfig, apps
 from django.conf import settings
 from django.core.exceptions import ImproperlyConfigured
+import importlib.metadata
```

Comment:
Please move this import line to the top. Separate import lines to 3 groups as [`PEP8` recommendation](https://peps.python.org/pep-0008/#imports).